docs/intent.html

```diff
@@ -54,9 +88,16 @@ <h5>Kotlin</h5>
 startActivity(intent)</pre>
 		<h5>HTML (Chrome)</h5>
 		<pre>&lt;a href="intent://example.org/media.png#Intent;type=video/any;package=is.xyz.mpv;scheme=https;end;"&gt;Click me&lt;/a&gt;</pre>
+		<pre>&lt;a href="intent:https://example.com/abc#Intent;package=is.xyz.mpv;S.title=abc;S.http-header-fields=Cookie: key1=val1%3b 
+key2=val2,Accept: */*,Accept-Language: en-RO%5c,en%3bq=0.9%5c,ro-RO%3bq=0.8%5c,ro%3bq=0.7%5c,en-US%3bq=0.6;end;"&gt;Click me&lt;/a&gt;</pre>
```

Comment:
spaces need to be encoded too

Reply:
Spaces need to be encoded as well but javascript already encodes spaces in the string when you assign it to a href attribute.
 I wanted to point out that you need to encode semicolons and backslashes yourself and the example felt too unreadable with spaces encoded as well